eggplant/market/templates/eggplant/market/checkout.html

```diff
@@ -2,63 +2,69 @@
 {% load bootstrap3 %}
 {% load i18n %}
 {% block content %}
-
-<div class="row">
-	<div class="col-md-12 col-span-3">
-		<h3 class="page-header">{% trans 'Your cart' %}</h3>
+	<div class="row">
+		<div class="col-md-12 col-span-3">
+			<h3 class="page-header">{% trans 'Your cart' %}</h3>
+		</div>
+	</div>
+{% if items %}
+	<div class="row">
+		<div class="col-md-3"><strong>Product</strong></div>
+		<div class="col-md-3"><strong>Delivery date</strong></div>
+		<div class="col-md-3"><strong>Delivered to:</strong></div>
+		<div class="col-md-1"><strong>Amount</strong></div>
+		<div class="col-md-2"><strong>Price</strong></div>
 	</div>
-</div>
-
-<div class="row">
-	<div class="col-md-3"><strong>Product</strong></div>
-	<div class="col-md-3"><strong>Delivery date</strong></div>
-	<div class="col-md-3"><strong>Delivered to:</strong></div>
-	<div class="col-md-3"><strong>Price</strong></div>
-</div>
-
-{% for item in items %}
-<div class="row">
-  <div class="col-md-3 text-left">{{item.product.title}}</div>
-  <div class="col-md-3">{{item.delivery_date|date:"d/m - Y"|default:"-"}}</div>
-  <div class="col-md-3">{{item.delivered_to}}</div>
-  <div class="col-md-3">{{item.product.price}} DKK</div>
-</div>
-{% endfor %}
 
-<div class="row">
-	<hr>
-	<div class="col-md-4 text-left">
-	<strong>Total</strong>
+	{% for item in items %}
+	<div class="row">
+	  <div class="col-md-3 text-left">{{item.product.title}}</div>
+	  <div class="col-md-3">{{item.delivery_date|date:"d/m - Y"|default:"-"}}</div>
+	  <div class="col-md-3">{{item.delivered_to}}</div>
+ 	  <div class="col-md-1">{{item.quantity}}</div>
+	  <div class="col-md-2">{{item.product.price}}</div>
 	</div>
-	<div class="col-md-4 text-right">
-		<strong>{{basket.get_total_amount}}</strong>
+	{% endfor %}
+
+	<div class="row">
+		<hr>
+		<div class="col-md-10 text-left">
+		<strong>Total</strong>
+		</div>
+		<div class="col-md-2">
+			<strong>{{basket.get_total_amount}}</strong>
+		</div>
 	</div>
-</div>
 
-<hr />
+	<hr />
+	<div class="row">
+		<div class="col-md-12 text-left">
+		<p>The above is your order. Please look it through it includes the items
```

Comment:
Well I didn’t create the most of the text and I hoped that someone is in charge of it. I just proposed in some cases how it could look like.